draft-ietf-rtcweb-jsep.xml

```diff
@@ -1097,6 +1097,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           simulcast as described in
           <xref target="sec.simulcast" />).</t>
         </section>
+
+        <section title="stopTransceiver" anchor="sec.transceiver-stop">
```

Comment:
That's a bit more remodeling of the text than I wanted to do.  Can we land this as-is and then do another PR to remodel?  Or would you like to make a PR of my PR?  I have a feeling you typing what you want will be faster than you typing a description of what you want me to type.